src/compiler/conversion/mod.rs

```diff
@@ -265,21 +267,26 @@ fn parse_timestamp(tz: TimeZone, s: &str) -> Result<DateTime<Utc>, Error> {
             return Ok(result);
         }
     }
-    for format in TIMESTAMP_UTC_FORMATS {
-        if let Ok(result) = Utc.datetime_from_str(s, format) {
-            return Ok(result);
-        }
-    }
+
+    // This also handles "%FT%TZ" formats.
     if let Ok(result) = DateTime::parse_from_rfc3339(s) {
         return Ok(datetime_to_utc(&result));
     }
+
     if let Ok(result) = DateTime::parse_from_rfc2822(s) {
         return Ok(datetime_to_utc(&result));
     }
+
     for format in TIMESTAMP_TZ_FORMATS {
         if let Ok(result) = DateTime::parse_from_str(s, format) {
             return Ok(datetime_to_utc(&result));
         }
     }
+
+    // This is equivalent to the "%s" format.
+    if let LocalResult::Single(result) = parse_unix_timestamp(s) {
+        return Ok(result);
+    }
```

Comment:
For maximum compatibility, this might need to move up after the first parse, since it would have been tried there previously. Maybe it doesn't matter, though, since the formats are not overlapping (i.e. if this would have succeeded, none of the others would have).

Reply:
Done. Ideally I would sort those parsing attempts by usage frequency.